app/components/DangerousWordInfo.tsx

```diff
@@ -0,0 +1,19 @@
+'use client';
+
+import { Icon, Tooltip, TooltipContent, TooltipTrigger } from '@swapr/ui';
+
+export const DangerousWordInfo = () => (
```

Comment:
I think, `DangerousWordsWarning`, describes better

```suggestion
export const DangerousWordsWarning = () => (
```